src/analysis/PointsTo/PointerAnalysis.h

```diff
@@ -93,6 +91,14 @@ class PointerAnalysis
 
     PointerSubgraph *getPS() const { return PS; }
 
+    bool isInLoop(PSNode *n) const {
```

Comment:
This does not look like a  method related to (arbitrary) pointer analysis, IMHO it should be moved to PointsToFlowSensitive.h